code/game/turfs/space/space.dm

```diff
@@ -15,6 +15,13 @@
 /turf/space/New()
 	if(!istype(src, /turf/space/transit))
 		icon_state = "[((x + y) ^ ~(x * y) + z) % 25]"
+	update_starlight()
+
+/turf/space/proc/update_starlight()
+	for(var/turf/T in range(src,1))
+		if(istype(T,/turf/simulated))
+			src.SetLuminosity(2)
+			return
 

```

Comment:
I don't really see how to handle that, unless you want me to call update starlight for any space tile in range when a turf is deconstructed (IE walls etc)